File: java-recipes/src/main/java/com/fargopolis/models/RecipeStep.java
package com.fargopolis.models;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class RecipeStep {
	private Integer stepId;
	private Integer recipeId;
	private Integer stepNumber;
	private String description;
}

File: recipe-site/app/_models/RecipeStep.ts
export default class RecipeStep {
    stepId: number;
    recipeId: number;
    stepNumber: number;
    description: string;

    constructor(
        stepId: number,
        recipeId: number,
        stepNumber: number,
        description: string
    ) {
        this.stepId = stepId;
        this.recipeId = recipeId;
        this.stepNumber = stepNumber;
        this.description = description;
    }

}